browser-test/fixtures/mock-server.js

```diff
@@ -0,0 +1,57 @@
+const execa = require('execa');
+const download = require('download');
+const fs = require('fs');
+const path = require('path');
+const rimraf = require('rimraf');
+const tar = require('tar');
+const util = require('util');
+
+const mkdir = util.promisify(fs.mkdir);
+const rmrf = util.promisify(rimraf);
+
+
+function sleep(timeout) {
+    return new Promise(resolve => {
+        setTimeout(resolve, timeout);
+    });
+}
+
+
+async function startServer() {
+    const testDir = path.join(process.cwd(), '.kitchen-sink');
+    const fallbackServerUrl = 'https://github.com/googleapis/gapic-showcase/releases/download/v0.2.4/gapic-showcase-0.2.4-linux-amd64.tar.gz'; 
+    const linuxTarFile = 'gapic-showcase-0.2.4-linux-amd64.tar.gz';
```

Comment:
I know this is merged, but I didn't catch this the first time. A separate PR would be welcome :smiley: 

Wherever there is a version used for a remote artifact, let's try to put it in 1 variable and make it configurable. Thus, when I release a new version of `gapic-showcase`, you don't have to find all of the places in the code where it is set and change it. 

1. I'd use a top-level variable (in the function or global, but probably not global if it's not used elsewhere) called `SHOWCASE_VERSION`
2. It should check an environment variable, maybe named `SHOWCASE_VERSION`, for a semver value (`0.3.7`)
3. If the environment variable is empty, use a default version, like `0.2.4`. 
4. the CI environment configuration should define this environment variable.

This also makes it easier to change things locally and reproduce bugs with tests.

Reply:
You're right! I also missed these three occurrences of `0.2.4`. I agree with the env var approach (`process.env['SHOWCASE_VERSION']`). I'll add this to the list of tasks.

cc: @ramya-ramalingam